packages/lib/server/repository/organization.ts

```diff
@@ -45,13 +45,13 @@ export class OrganizationRepository {
             isOrganizationVerified: true,
             isOrganizationConfigured: orgData.isOrganizationConfigured,
             orgAutoAcceptEmail: orgData.autoAcceptEmail,
-            isPlatform: orgData.isPlatform,
           },
         },
         metadata: {
           ...(IS_TEAM_BILLING_ENABLED ? { requestedSlug: orgData.slug } : {}),
           orgSeats: orgData.seats,
           orgPricePerSeat: orgData.pricePerSeat,
+          isPlatform: orgData.isPlatform,
```

Comment:
@zomars Why was this moved to metadata here? There is a standalone column `isPlatform` on the team table that the platform team has intended to use.